js-library/src/contracts/FDSNameResolver/FDSNameResolver.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
+  "_format": "hh-sol-dbg-1",
+  "buildInfo": "../../build-info/041b9b8ba8f9846aee37e132a76f9297.json"
```

Comment:
what file does it refer to? It has not been added to the project

Reply:
These files are generated by hardhat when contracts are compiled. Maybe we don't need the dbg.json files, but the other json contain abi section that is required.